CHANGELOG.md

```diff
@@ -12,6 +12,10 @@ how a consumer would use the library (e.g. adding unit tests, updating documenta
 
 ### Changed
 
+- The `--advanced-query` option on `alerts search` and `security-data (search|send-to)` commands has been updated:
```

Comment:
we should call out the `@` prefix here as well as in the help documentation string.

Reply:
done